app/assets/javascripts/select-kit/addon/components/email-group-user-chooser-header.js

```diff
@@ -0,0 +1,80 @@
+import MultiSelectHeaderComponent from "select-kit/components/multi-select/multi-select-header";
+import { computed } from "@ember/object";
+import { gt } from "@ember/object/computed";
+import { isTesting } from "discourse-common/config/environment";
+import layout from "select-kit/templates/components/email-group-user-chooser-header";
+
+export default MultiSelectHeaderComponent.extend({
+  layout,
+  classNames: ["email-group-user-chooser-header"],
+  hasHiddenItems: gt("hiddenItemsCount", 0),
+
+  shownItems: computed("hiddenItemsCount", function () {
+    if (
+      this.selectKit.noneItem === this.selectedContent ||
+      this.hiddenItemsCount === 0
+    ) {
+      return this.selectedContent;
+    }
+    return this.selectedContent.slice(
+      0,
+      this.selectedContent.length - this.hiddenItemsCount
+    );
+  }),
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.set("__isRendered", true);
```

Comment:
can you explain why you need this isRendered ?

Reply:
I removed it.